cinema/views.py

```diff
@@ -43,6 +48,21 @@
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+        if actors:
+            actors_id = [int(str_id) for str_id in actors.split(",")]
+            queryset = queryset.filter(actors__id__in=actors_id)
+        if genres:
+            genres_ids = [int(str_id) for str_id in genres.split(",")]
```

Comment:
Similarly, ensure that the conversion of genre IDs to integers in `get_queryset` handles potential ValueErrors gracefully.